coriolis_openstack_utils/security_groups.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from oslo_log import log as logging
+
+
+LOG = logging.getLogger(__name__)
+
+
+def list_security_groups(openstack_client, tenant_id, secgroup_name=None):
+    if secgroup_name is None:
+        if int(openstack_client.connection_info["identity_api_version"]) == 2:
+            return openstack_client.neutron.list_security_groups(
+                tenant_id=tenant_id)['security_groups']
+        return openstack_client.neutron.list_security_groups(
+            project_id=tenant_id)['security_groups']
+    else:
+        if int(openstack_client.connection_info["identity_api_version"]) == 2:
+            return openstack_client.neutron.list_security_groups(
+                tenant_id=tenant_id, name=secgroup_name)['security_groups']
+        return openstack_client.neutron.list_security_groups(
+            project_id=tenant_id, name=secgroup_name)['security_groups']
+
+
+def create_security_group(openstack_client, tenant_id, body):
+    if int(openstack_client.connection_info["identity_api_version"]) == 2:
+        body['tenant_id'] = tenant_id
```

Comment:
Why not just put both `tenant_id` and `project_id` in the secgroup body? (afaik `tenant_id` should have been kept for backwards-compatibility, but the Neutron API should ignore extra params even if it wasn't)